packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -74,7 +74,10 @@ if (!ShadowRoot.prototype.createElement) {
       // Register a stand-in class which will handle the registry lookup & delegation
       let standInClass = nativeGet.call(nativeRegistry, tagName);
       if (!standInClass) {
-        standInClass = createStandInElement(tagName);
+        standInClass = createStandInElement(
+          tagName,
+          elementClass.formAssociated
```

Comment:
So the problem with the `standInElement` is that only the settings on the first element registered will be respected by the browser.

For callbacks, the `standInElement` registers generic callbacks that can then delegate at runtime to the class for the scope it's used in. However, it's harder for static settings that can't be delegated at runtime. So for `formAssociated`, if the first class that happens to be registered for that tag name has `formAssociated: false`, any subsequent ones won't be form-associated.

There's a similar problem with `observedAttributes`, but we're able to mostly work around that by patching `setAttribute` to come close to mimicking the browser behavior for the per-class `observedAttributes` settings. It seems like we'd need to default this to `true` for all classes, but I don't know all the implications of that or if there's a way at runtime to make a `formAssociated: true` element behave as if it wasn't form associated; that would need some thought, and if it's not possible, it might just be a hard limitation of the polyfill.

Reply:
re: callbacks, does that mean we'd need to add something to watch for the new callbacks like `formAssociatedCallback`, `formResetCallback`, etc?